Documentation/Signoffs/Pneumatic Road Tube System.md

```diff
@@ -0,0 +1,48 @@
+<h1>Detailed Design for Pneumatic Road Pressure Tube Subsystem within the Secondary Data Acquisition System</h1>
+
+
+<h2>Big Picture</h2>
+
+This subsystem will be responsible for counting how many vehicles are leaving or entering the Bell Hall parking lot. Counting will be done by pneumatic road tube sensors that will transmit their data to a counter. The counter will then transmit the data wirelessly to the server for processing.
+
+<h3>Specifications</h3>
+
+1. Selected Sensor Specifications
+  *Pressure Range: 0 to 50kPa
+  * 3 Vdc power needed
+  * Microprocessor to process sensor data
+
+2. Wireless Data Transfer Module
+
+3. Outdoor Case & Hardware
+
+<h4>Analysis</h4>
+
+  *Selected Sensor Specification
+
+ The MX53DP pressure sensor[1] needs to be able to detect variations in pressure inside the road tube. After researching different types of sensors, differential pressure sensing seems like the best solution. Using the differential pressure sensor, how much absolute pressure does not need to be known. Only the difference between two point, which will be the pressure in the tube and the internal vacuum pressure inside the sensor. The Arduino UNO R3[2] board will use Smakn 8194[3], which is a 12Vdc to 9Vdc step down converter to power it. The sensor however needs 3Vdc to power it so Digi-Key LD1085V[4], a 3 pin IC voltage regulator will be used to acquire the needed voltage. The sensor signal pin (2) will be connected to the IO pins on an Arduino Uno board to signify when the pressure in the road tube has exceeded the internal vacuum pressure of the sensor. 
+
+
+  *Wireless Data Transfer Module
+
+Data will be transmitted wirelessly back to the server using a ESP8288 Wi-Fi module[5]. This component works especially well with Arduino devices. 
```

Comment:
State the requirements. Not components. You can't make decisions without clearly understanding the requirements. How far, how much data etc.